pyatlan/model/packages/asset_export(basic).py

```diff
@@ -0,0 +1,290 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from pyatlan.model.enums import CustomWorkflowPackage
+from pyatlan.model.packages.base.custom_package import AbstractCustomPackage
+from pyatlan.model.workflow import WorkflowMetadata
+
+
+class AssetExportPackage(AbstractCustomPackage):
```

Comment:
I think we should rename this to something shorter by dropping `_Package`? for ex: `AssetExport`, `AssetImport`, or `RelationalAssetBuilder`.

Reply:
It’s reasonable to shorten it and avoid stating the obvious, but aren't we also using this as a convention for crawlers? For example, `ConfluentKafkaCrawler` and `PowerBICrawler`.

Reply:
Okay, let's keep it as it is with `_Package` for now!

Reply:
I found a utility package, [`ConnectionDelete`](https://github.com/atlanhq/atlan-python/blob/main/pyatlan/model/packages/connection_delete.py), which was released a long time ago. I think for these types of custom packages, we can avoid adding an additional suffix (`_Package`). What do you think?

Reply:
If we follow this: https://github.com/atlanhq/atlan-python/pull/402#discussion_r1818828755, then the class name would be `AssetExportBasic`.